build.gradle

```diff
@@ -49,6 +49,15 @@ repositories {
     mavenCentral()
     jcenter()
     mavenLocal()
+    repositories {
+        maven {
+            url = uri("https://maven.pkg.github.com/FINTLabs/fint-arkiv-case-defaults")
+            credentials {
+                username = project.findProperty("gpr.user") ?: System.getenv("USERNAME")
+                password = project.findProperty("gpr.key") ?: System.getenv("TOKEN")
+            }
+        }
+    }
```

Comment:
Any reason not to deploy the latest and greatest fint-arkiv-case-defaults to repo.fintlabs.no?

Reply:
Nope, no reason. :)